mne/preprocessing/maxwell.py

```diff
@@ -1975,6 +1977,18 @@ def find_bad_channels_maxwell(
 
     .. versionadded:: 0.20
     """
+    if h_freq is not None:
+        if 'lowpass' in raw.info:
```

Comment:
lowpass is always present but it can be None. I think you will warn all the time here.

What I would do is something like this:

`if raw.info.get('lowpass') and raw.info['lowpass'] < 'lowpass' `

Reply:
I've changed things in 94e5807, can you have a look?